pyvrp/tests/test_Individual.py

```diff
@@ -322,10 +322,6 @@ def test_str_contains_essential_information():
             for client in route:
                 assert_(str(client) in str_route)
 
-        # Last lines should contain the travel distance and collected prizes.
```

Comment:
I'm pretty sure the asserts above in this test now fail (because + 2 is no longer needed). Can we update the comments/asserts there as well?

Reply:
Maybe also rename the test to `test_str_contains_routes` or similar?

Reply:
Sorry, that was sloppy 😅